.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+:memo: :lipstick: :new: :bomb: :bug: :art: Add relevant emojis to title. [See style guide.](https://github.com/jameshughes89/cs102/blob/contribute/CONTRIBUTING.md#style-guidelines)
+:memo: :lipstick: :new: :bomb: :bug: :art: Replace this line with only relevant emojis
```

Comment:
These are duplicated now.

Reply:
Was deliberately designed this way, but on further thought, no need for the emojis in body of the issue/PR.

Reply:
That's more or less what labels are for. Labels are also searchable.

Reply:
But, I think labels are add-able by people with write access to the repo. Might be settings that you can change for this? Not sure.